src/chat/tools/base.ts

```diff
@@ -19,8 +19,6 @@ import { LanguageModelToolContribution } from "./types";
  */
 export abstract class BaseLanguageModelTool<T> implements LanguageModelTool<T> {
   abstract readonly name: string;
-  /** Message to be shown when this tool is called in a chat session via `stream.progress()`. */
-  abstract readonly progressMessage: string;
```

Comment:
Don't forget to remove the `readonly progressMessage` bits from the existing tools as well!